compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/useMemo-multiple-returns.js

```diff
@@ -0,0 +1,9 @@
+function Component({items}) {
+  const value = useMemo(() => {
+    for (let item of items) {
+      if (item.match) return item;
+    }
+    return null;
```

Comment:
Leaving this out seems kinda fine and more stylistic. Should we lint for at least one non-void return? Consistent return I think already exists as an independent lint outside React's linters.